pypose/lietensor/lietensor.py

```diff
@@ -1108,7 +1108,7 @@ def euler(self, eps=2e-4) -> torch.Tensor:
         roll2 = torch.zeros_like(t0)
         flag = t2.abs() < 1. - eps
         yaw1 = torch.atan2(t3, t4)
-        yaw2 = -2 * torch.sign(t2) * torch.atan2(x, w)
+        yaw2 = -2 * pm(t2) * torch.atan2(x, w)
```

Comment:
@haleqiu  please have a look at this change. Could you merge this PR, if it meets your expectation?